tests/infer/mcmc/test_nuts.py

```diff
@@ -279,7 +285,7 @@ def model(data):
     assert_equal(posterior.mean, y_prob, prec=0.05)
 
 
-@pytest.mark.parametrize("jit", [False, mark_jit(True)], ids=jit_idfn)
+@pytest.mark.parametrize("jit", [True], ids=jit_idfn)
```

Comment:
I don't see any test which is slower with `jit_compile=True`. And this is the slowest test on Travis, so we can put a test for jit here.